Automattic/jetpack
# Social Logos: Rewrite svg-optimize script in JS

As the title says, this rewrites said script in JS.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Make sure `./tools/build` continues to create optimized SVGs in `./build/svg-clean` as before.
